api/src/models/user/user.methods.ts

```diff
@@ -114,6 +114,8 @@ export async function addConnectionToUser (this: IUserDocument, objId: string, i
       this["connections"][targetUser._id] = targetUserConnection;
       targetUser["connectionOf"][this._id] = originatorConnection;
 
+      this.markModified("connections");
+      targetUser.markModified("connectionOf");
```

Comment:
Can we paste these below for the delete method too? deleteConnectionFromUser currently doesn't work. I guess maybe it would be worth getting the document again after the save to test if the operations worked, then it won't return 200 if it didn't work.